Classes/Domain/Model/Flexform.php

```diff
@@ -45,6 +45,13 @@ class Flexform extends \TYPO3\CMS\Extbase\DomainObject\AbstractValueObject
      */
     protected $cLang;
 
+    /**
+     * Code URL / File
+     *
+     * @var string
+     */
+    protected $cFile;
```

Comment:
I'm not that happy with naming this variable `file` when it's supposed to hold a local file OR an URL. What renaming this to `$cUri`? Sounds more fitting, don't you think?